### ISSUE TITLE
[Warlock Imp] Walks in after each firebolt cast.  Low mana regen.

### ISSUE THREAD

**Issue by Garrus71** [dalaranwow/dalaran-wow#3959]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Warlock Imp pet taking steps forward after firebolt cast and mana return is not where it should be.

**Expected behaviour**: Stay at max range and be able to cast indefinitely in 25m.

**Steps to reproduce the problem**:

1.  Stand next to high hp target at max range
2.  /petattack

**Include proofs for this behaviour**

Replicated 100% of the time

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** NA

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary** NA

**Comment by Rushor** [comment]
first fixed in https://github.com/dalaranwow/dalaran-wow/issues/1375 (need to be tested on PTR)

second duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3216
